### ISSUE TITLE
cross build to 2.11? 

### ISSUE THREAD

**Issue by kailuowang** [jmcardon/tsec#116]
Is there any plan to cross build to scala 2.11? If so I can submit a PR.

**Comment by kailuowang** [comment]
also would you consider publish to sonatype so it's available on the default maven central?

**Comment by jmcardon** [comment]
- In regards to sonatype, if someone wants to put in the effort sure. I've been quite busy recently and  dealing with publishing to sonatype and all the complications isn't something I'm working on.
- 2.11? Why not? Feel free to submit a PR @kailuowang! Thanks for the interest in the project, sorry for the slow publishing recently. Both myself and the maintainers have been somewhat busy.

**Comment by kailuowang** [comment]
Yeah sonatype is quite a hassle to setup especially if you haven't setup an account with them. And it's not something someone else could do for you. 
As for 2.11 I realized that you are not using sbt-release? it would be a lot easier to do release with that.

**Comment by jmcardon** [comment]
@kailuowang Updating this ticket, myself and @ChristopherDavenport are working on it for next release.

Sonatype org repo is up, just gotta configure. Thanks for the suggestion. I needed a nudge.

**Comment by kailuowang** [comment]
Thanks a lot @jmcardon and @ChristopherDavenport
